crates/parcel_napi_helpers/src/js_callable/js_callable.rs

```diff
@@ -58,6 +58,11 @@ impl JsCallable {
     Self::new(jsfn)
   }
 
+  pub fn into_unref(mut self, env: &Env) -> napi::Result<Self> {
+    self.threadsafe_function.unref(env)?;
```

Comment:
I don't think this is right, we should do it on drop if we need to

Reply:
https://nodejs.org/api/n-api.html#napi_unref_threadsafe_function

Nevermind, I guess this is okay

Reply:
:facepalm:  Thanks